From d4eb78ea3aec45dffa2ee78d8128e46d9dae4fb5 Mon Sep 17 00:00:00 2001
From: stack <stack@apache.org>
Date: Thu, 4 Feb 2016 16:10:53 -0800
Subject: [PATCH 061/217] Revert "HBASE-15120 Undo aggressive load balancer
 logging at tens of lines per millisecond"

This reverts commit 69b70b87aa2ddd808623912d93d627fe02e211f4.
---
 .../hbase/master/balancer/BaseLoadBalancer.java    |    8 ++------
 1 file changed, 2 insertions(+), 6 deletions(-)

diff --git a/hbase-server/src/main/java/org/apache/hadoop/hbase/master/balancer/BaseLoadBalancer.java b/hbase-server/src/main/java/org/apache/hadoop/hbase/master/balancer/BaseLoadBalancer.java
index fd1ddd8..8dc7cf6 100644
--- a/hbase-server/src/main/java/org/apache/hadoop/hbase/master/balancer/BaseLoadBalancer.java
+++ b/hbase-server/src/main/java/org/apache/hadoop/hbase/master/balancer/BaseLoadBalancer.java
@@ -799,11 +799,9 @@ public abstract class BaseLoadBalancer implements LoadBalancer {
           i++;
           lowestLocalityServerIndex = serverIndicesSortedByLocality[i];
         }
-        if (LOG.isTraceEnabled()) {
-          LOG.trace("Lowest locality region server with non zero regions is "
+        LOG.debug("Lowest locality region server with non zero regions is "
             + servers[lowestLocalityServerIndex].getHostname() + " with locality "
             + localityPerServer[lowestLocalityServerIndex]);
-        }
         return lowestLocalityServerIndex;
       }
     }
@@ -826,11 +824,9 @@ public abstract class BaseLoadBalancer implements LoadBalancer {
             lowestLocalityRegionIndex = j;
           }
         }
-        if (LOG.isTraceEnabled()) {
-          LOG.debug(" Lowest locality region index is " + lowestLocalityRegionIndex
+        LOG.debug(" Lowest locality region index is " + lowestLocalityRegionIndex
             + " and its region server contains " + regionsPerServer[serverIndex].length
             + " regions");
-        }
         return regionsPerServer[serverIndex][lowestLocalityRegionIndex];
       } else {
         return -1;
-- 
1.7.9.5

